frontend/amundsen_application/static/js/pages/TableDetailPage/index.tsx

```diff
@@ -533,7 +534,7 @@ export class TableDetail extends React.Component<
                 <section className="right-panel">
                   <EditableSection
                     title={Constants.OWNERS_TITLE}
-                    readOnly={!data.is_editable}
+                    readOnly
```

Comment:
make owners change readOnly

Reply:
the Owners doesn't seem to be editable now, is it?